### DISCUSSION TITLE
Images and paths are not working properly in the deployed page

### DISCUSSION THREAD

**Discussion by kristosh** [alshedivat/al-folio#1401] [Q&A]
Hi after following the steps for deploying the page online I have successfully managed to deploy in the following [link](https://kristosh.github.io/kristosh/). However, my issue is that the paths are not recognized properly (while it looks fine locally) and the page looks a bit weird (a little bit drunk ;)). 

Is that a common issue? Do I need to perform any extra updates to make all the paths work when deploying the page online? 

Literally, everything else seems to work properly except the paths :(

**Comment by george-gca** [comment]
I was organizing the discussions in our repo and just found your issue. Hope that it was fixed.
